test/signer.test.ts

```diff
@@ -60,4 +60,24 @@ describe("BLS Signer", async () => {
             assert.isTrue(signer.verify(signature, signer.pubkey, message));
         }
     }).timeout(20000);
+
+    describe("getSigner", () => {
+        let factory: BlsSignerFactory;
+        before(async () => {
+            factory = await BlsSignerFactory.new();
+        });
+
+        [
+            "",
```

Comment:
This parses in to `mcl` correctly, so I assume it is a valid secret. However, this could be passed in if a developer/user makes a mistake. Do we want to throw if the secret passed in is empty?

Reply:
0 is a valid value for a secret so leave as is for now.